acct_mgt_test.py

```diff
@@ -247,136 +245,127 @@ def ms_check_user(acct_mgt_url, user_name, auth_opts=[]):
         + auth_opts
         + [acct_mgt_url + "/users/" + user_name]
     )
-    pprint.pprint(cmd)
     result = subprocess.run(
         cmd,
         stdout=subprocess.PIPE,
         stderr=subprocess.STDOUT,
+        check=False,
     )
     return compare_results(result, r'{"msg": "user \(' + user_name + r'\) exists"}')
 
 
-def ms_create_user(acct_mgt_url, user_name, auth_opts=[]):
-
+def ms_create_user(acct_mgt_url, user_name, auth_opts=None):
+    """Creates a user using the microserver"""
     result = subprocess.run(
         ["curl", "-X", "PUT", "-kv"]
         + auth_opts
         + [acct_mgt_url + "/users/" + user_name],
         stdout=subprocess.PIPE,
         stderr=subprocess.STDOUT,
+        check=False,
     )
     return compare_results(result, r'{"msg": "user created \(' + user_name + r'\)"}')
 
 
-def ms_delete_user(acct_mgt_url, user_name, auth_opts=[]):
+def ms_delete_user(acct_mgt_url, user_name, auth_opts=None):
+    """Deletes a user usering the microserver"""
     cmd = (
         ["curl", "-X", "DELETE", "-kv"]
         + auth_opts
-        + [acct_mgt_url + "/users/" + user_name]
+        + [f"{acct_mgt_url}/users/{user_name}"]
     )
-    pprint.pprint(cmd)
     result = subprocess.run(
         cmd,
         stdout=subprocess.PIPE,
         stderr=subprocess.STDOUT,
+        check=False,
     )
     return compare_results(result, r'{"msg": "user deleted \(' + user_name + r'\)"}')
 
 
 def ms_user_project_get_role(
     acct_mgt_url,
-    user_name,
-    project_name,
-    role,
+    role_info,
     success_pattern,
-    auth_opts=[],
```

Comment:
Ah, nice catch from the linter. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments